cinema/tests/test_movie_and_movie_session.py

```diff
@@ -0,0 +1,286 @@
+
+
+class MovieAndMovieSessionTestCaseForAnonimUser(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+
+    def test_movie_if_anonim_user(self):
+        movie_response = self.client.get(MOVIE_LIST_URL)
+
+        self.assertEqual(movie_response.status_code, status.HTTP_401_UNAUTHORIZED)
+
+    def test_movie_sessions_if_anonim_user(self):
+        movie_sessions_response = self.client.get(MOVIE_SESSION_LIST_URL)
+
+        self.assertEqual(
+            movie_sessions_response.status_code, status.HTTP_401_UNAUTHORIZED
+        )
+
+
+class MovieAndMovieSessionTestCaseForSimpleUser(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+        self.user = User.objects.create_user(
+            email="<EMAIL>",
+            password="<PASSWORD>",
+        )
+        self.client.force_authenticate(user=self.user)
+
+        self.actor_John = Actor.objects.create(
+            first_name="John",
+            last_name="Doe",
+        )
+        self.actor_Illia = Actor.objects.create(
+            first_name="illia",
+            last_name="Doe",
+        )
+
+        self.genre_horror = Genre.objects.create(name="horror")
+        self.genre_drama = Genre.objects.create(name="drama")
+
+        self.cinema_hall = CinemaHall.objects.create(
+            name="Cinema Hall",
+            rows=12,
+            seats_in_row=4,
+        )
+        self.movie_vinston = Movie.objects.create(
+            title="vinston",
+            description="Test movie",
+            duration=6,
+        )
+        self.movie_apalon = Movie.objects.create(
+            title="apalon",
+            description="Test movie",
+            duration=6,
+        )
+        self.movie_japon = Movie.objects.create(
+            title="japon",
+            description="Test movie",
+            duration=6,
+        )
+
+        self.movie_apalon.actors.add(self.actor_John)
+        self.movie_apalon.genres.add(self.genre_horror)
+
+        self.movie_vinston.actors.add(self.actor_Illia)
+        self.movie_vinston.genres.add(self.genre_drama)
+
+        self.movie_japon.actors.add(self.actor_John)
+        self.movie_japon.genres.add(self.genre_drama)
+
+        self.movie_session_2024 = MovieSession.objects.create(
+            show_time=datetime.datetime(2024, 1, 1, 12, 0, 0),
+            movie=self.movie_apalon,
+            cinema_hall=self.cinema_hall,
+        )
+        self.movie_session_2023 = MovieSession.objects.create(
+            show_time=datetime.datetime(2023, 1, 1, 12, 0, 0),
+            movie=self.movie_vinston,
+            cinema_hall=self.cinema_hall,
+        )
+
+    def test_movie_if_simple_user(self):
+        movie_response = self.client.get(MOVIE_LIST_URL)
+
+        movies = Movie.objects.all()
+
+        movie_serializer = MovieListSerializer(movies, many=True)
+
+        self.assertEqual(movie_response.status_code, status.HTTP_200_OK)
+
+        self.assertEqual(movie_serializer.data, movie_response.data)
+
```

Comment:
This test could be improved by adding assertions for the specific fields in the response. This would make your test more robust and likely to catch regressions.